src/components/modules/shortcuts.ts

```diff
@@ -71,4 +71,14 @@ export default class Shortcuts extends Module {
     this.registeredShortcuts[index].remove();
     this.registeredShortcuts.splice(index, 1);
   }
+
+  /**
+   * Module destruction
+   * De-registers all Shortcuts
+   */
+  public destroy(): void {
+    while (this.registeredShortcuts.length > 0) {
```

Comment:
Wouldn't this while loop run infinitely if for some reason the remove function returns from line 68? There is no reason for that to happen but just in case. A for of loop would be safer?

Reply:
Sorry for my delay. 

I think when line 68 removed occurs more other bugs. Even use for loop instead of while, it is not a resolution of true problem.